### PR TITLE
LPS-69699 Alloyeditor placeholders

### PR DESCRIPTION
**Pull request by ambrinchaudhary**

### CODE REVIEW

**Comment by ambrinchaudhary** [comment]
cc @sergiogonzalez @antoniopol06

File: modules/apps/collaboration/blogs/blogs-web/src/main/resources/META-INF/resources/blogs/css/main.scss [hunk 1]
```diff
@@ -148,6 +148,12 @@
 	}
 
 	.edit-entry {
+		.alloy-editor-placeholder:before {
```
Comment (marcoscv-work) [inline modules/apps/collaboration/blogs/blogs-web/src/main/resources/META-INF/resources/blogs/css/main.scss]: Hey Ambrin!

Try with this:

.ae-placeholder:not(:focus):before {
   content: attr(data-placeholder);
   position: absolute;
   transform: translateX(-50%);
}

Comment (ambrinchaudhary) [inline modules/apps/collaboration/blogs/blogs-web/src/main/resources/META-INF/resources/blogs/css/main.scss]: That would center every placeholder we have in Blogs, and some of them shouldn't. This way we inherit the parent text-aligment.

**Comment by brianchandotcom** [comment]
Merged. Thx.

**Comment by brianchandotcom** [comment]
@marcoscv-work I merged this. If you don't agree, please send a pull to @ambrinchaudhary thx.

**Pull request closed without merging**